File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.cpp
class Solution {
public:
    string removeOuterParentheses(string s) {
        string res="";
        int cnt=0;
        for(int i=0;i<s.length();i++)
        {
            if(s[i]=='(')
            {
                if(cnt!=0)
                {
                    res = res+s[i];
                }
                cnt++;
            }
            else if(s[i]==')')
            {
                if(cnt!=1)
                {
                    res = res+s[i];
                }
                cnt--;
            }
        }
        return res;
    }
};

File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.py
class Solution:
    def removeOuterParentheses(self, s: str) -> str:
        cnt=0
        res=''
        for i in s:
            if i=='(':
                if cnt!=0:
                    res+=i
                cnt+=1
            elif i==')':
                if cnt!=1:
                    res+=i
                cnt-=1
        print(cnt)
        return res